### DISCUSSION TITLE
Make DrawingView (and other components) work in .razor (maui blazor) files

### DISCUSSION THREAD

**Discussion by erikruniaOM** [CommunityToolkit/Maui#1112] [New Feature Discussions]
I've tried to get the DrawingView to work when using Maui Blazor in a .razor file, but with no luck.  Is there a plan to support CommunityToolkit outside of XAML pages?

**Comment by VladislavAntonyuk** [comment]
I would say no. 
You can’t use MAUI controls in html
Use Blazor specific SignaturePad control.

**Reply by erikruniaOM** [reply]
I don't need a signature pad, I need a drawing canvas.  Is there a blazor specific drawing canvas?

**Reply by VladislavAntonyuk** [reply]
yes, extension is called SignaturePad. https://github.com/MarvinKlein1508/SignaturePad
